docs/docs/markers.mdx

```diff
@@ -0,0 +1,291 @@
+---
+id: markers
+sidebar_label: Markers
+title: Markers
+description: Read more about how to mark points of interest in dialogues using logical expressions.
+abstract: Markers are logical expressions that allow you to describe points in the dialogue that you are interested in. 
+---
+
+:::caution
+
+This feature is currently experimental and might change or be removed in the future. Share your feedback in the forum to help us make it production-ready.
+
+:::
+
+Markers allow you to describe points in a dialogue that you are interested in identifying. In Rasa, a dialogue is represented as a sequence of events, which include bot actions that were executed (`ActionExecuted`), intents that were detected (`UserUttered`), and slots that were set (`SlotSet`). A point of interest in a dialogue can thus be expressed as a logical expression that can be evaluated against the sequence of dialogue events. When this logical expression is met, we "mark" that point in the dialogue for further analysis or inspection.
```

Comment:
This is a fairly low-level technical description - would it make sense to have something like a 3-column figure, where in the first column we show a bunch of turns in a carbon-bot conversation, in the middle column we show the technical view on how this is translated to Rasa's event implementation and in the 3rd column we show how the success-markers steedmark a particular point in the dialogue on the basis of a yaml config and the current tracker state.

Reply:
Thomas the figure is a good idea but requires some thought. I'll reserve it for the blog post 🙂